### ISSUE TITLE
Doctests failing since the deprecation of old documentation format (elixir 1.7)

### ISSUE THREAD

**Issue by johantell** [antonmi/espec#276]
Since [`Code.get_docs/2`](https://hexdocs.pm/elixir/Code.html#get_docs/2) is deprecated and returns `nil` whenever called in elixir 1.7+ all doc tests will fail with the error

```
  ** (ESpec.DocExample.Error) could not retrieve the documentation for module [MODULE_NAME].
 The module was not compiled with documentation or its beam file cannot be accessed
```

In order to support doctests ESpec need to use the newer format retrieved via [`Code.fetch_docs/1`](https://hexdocs.pm/elixir/Code.html#fetch_docs/1) when elixir 1.7+ is used.

**Comment by antonmi** [comment]
Thanks, @johantell !
Will fix it!

**Comment by antonmi** [comment]
Hi, @johantell !
The issue is fixed in 1.6.1.
Please check it out!

**Comment by johantell** [comment]
@antonmi works like a charm. Good work! 💛💚💙
